eng/Version.Details.xml

```diff
@@ -64,6 +64,10 @@
       <Uri>https://github.com/aspnet/AspNetCore</Uri>
       <Sha>1937c804fa83ee3adddc7e648c610f880c24188e</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.3.1" Version="1.0.2-beta4.19465.1" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
```

Comment:
Not feedback for PR, buit just making a note that we still need to wire up the subscription here, and capturing the asset/channel info.

```
λ darc get-asset --name "Microsoft.DotNet.Test.ProjectTemplates.3.1"
Looking up assets with name 'Microsoft.DotNet.Test.ProjectTemplates.3.1' in the last 30 days
Microsoft.DotNet.Test.ProjectTemplates.3.1 @ 1.0.2-beta4.19465.1
Repository: https://github.com/dotnet/test-templates
Branch: refs/heads/master
Commit: 6ae8a6e7dd862c9447771c108d43fb9028f22c22
Build Number: 20190915.1
Date Produced: 9/15/2019 10:07 PM
Build Link: https://dev.azure.com/dnceng/internal/_build/results?buildId=356021
BAR Build Id: 28417
Channels:
- .NET Core 3 Release
```

All of the templates from this repo come from a single branch so we need a .NET Core 3 Release branch subscription going to 3.1, which is a bit confusing.